elasticdl/python/master/task_dispatcher.py

```diff
@@ -49,6 +50,8 @@ def __init__(
             records_per_task: The number of records per task.
             num_epochs: The total number of epochs for the tasks where
                 an epoch is a complete iteration over the shards.
+            need_save_model: Save model or not after all the
+                training/evaluation/prediction task are completed.
```

Comment:
`Whether to save model after all tasks complete`

Reply:
Good suggestion, updated.